Social sharing icons

mnimmz opened the pull request:
Created social sharing icons for events page, news page, and photostream/photo-gallery images.

mnimmz commented:
Yes, the differences were intentional. 

1. /sharer/sharer.php? at one point was removed from Facebook to be used and they brought it back without announcing it but they prefer sharing links to use /dialogue/share? instead.

2. The app id needs to be included when using the /dialogue/share? method.

3. Incorporating bitly with social sharing buttons to shorten links seems to be only possible if we paste the sharing link into bitly directly. I attempted to use bitly's api for sharing links we manually build but I believe it's only supposed to shorten the link with an 'onclick' function, as that's what happened for me. The link would not appear shortened when it appeared in Twitter, Linkedin or Facebook's dialogue boxes to share it. Twitter automatically shortens links if it's more than 20 characters. I don't believe Facebook or Linkedin has this feature from what I saw.

raptureaid commented:
I've added this to staging and reviewed with Jamie. Twitter and LinkedIn appeared to work, but we were having issues with Facebook.

mnimmz commented:
I believe I fixed the issue with the Facebook link sharing Google's URL. I was playing around with Open Graph meta tags and I used google's url as a placeholder. I removed it so it should point to the correct url now.
